plugins/ssr/ssrelim.ml

```diff
@@ -185,9 +186,12 @@ let ssrelim ?(ind=ref None) ?(is_case=false) deps what ?elim eqid elim_intro_tac
         analyze_eliminator elimty env (project gl) in
       if is_case then
```

Comment:
I think this whole block (the two branches of `is_case`) deserves a comment explaining what it does.